frontend/src/utils/case.ts

```diff
@@ -0,0 +1,100 @@
+// Copied from https://github.com/blakeembrey/change-case
```

Comment:
Do you mind adding an explanation here as to why it was copied, similar to what's in the issue description?